testbed/tests/test_window.py

```diff
@@ -487,6 +487,21 @@ async def test_as_image(main_window, main_window_probe):
     main_window_probe.assert_image_size(screenshot.size, main_window_probe.content_size)
 
 
+# Test the `origin`, `position` and `screen_position`.
+async def test_screen(main_window, main_window_probe):
+    """The window can be relocated to another screen, using both absolute and relative screen positions."""
+
+    if main_window.screen.origin == (0, 0):
```

Comment:
Rather than skipping the test if the origin isn't 0,0, it would be better to make the test adapt to *any* origin.

Reply:
Done.